holmes/checks/checks_api.py

```
@@ -0,0 +1,214 @@
+            message=result.message,
+            duration=result.duration,
+            rationale=result.rationale,
+            error=result.error,
+            model_used=ai.llm.model,  # Include the actual model that was used
+            notifications=(
+                notifications if notifications else None
+            ),  # Include notification statuses
+        )
+
+    except AuthenticationError as e:
+        raise HTTPException(status_code=401, detail=e.message)
```

Ruff (0.14.13)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)